test/integration/Euler.t.sol

```diff
@@ -13,7 +13,7 @@ import { IRewardsController } from "src/interfaces/aave/IRewardsController.sol";
 
 import { FactoryStoreLib } from "src/libraries/FactoryStore.sol";
```

Comment:
why so many lines deleted in tests, do we still have sufficient coverage?

Reply:
achieved 100% of the asset manager code: 

https://app.codecov.io/gh/reservoir-labs/amm-core/blob/feat%2Fsupply-euler-vaults/src%2Fasset-management%2FEulerV2Manager.sol